sessions/git-and-github.qmd

```diff
@@ -1 +1,42 @@
-# Review of Git and GitHub {#sec-git-and-github}
\ No newline at end of file
+# Review of Git and GitHub {#sec-git-and-github}
+
+<!-- TODO: Add an introduction here -->
+
+## Learning objectives
+
+The **overall learning** outcome of this session is to:
+
+1.  Outline the core concepts and functionalities of Git and GitHub and
+    understand their role in facilitating effective collaboration
+
+1.  
+
+Specific **objectives** are to:
+
+1.  Explain the concept of "formal" version control and its key benefits
+2.  Describe the main features of Git and GitHub, including:
+    -   Local and remote repositories
+    -   Branching and its role in parallel work on the same project
+    -   Committing changes to track file history
+    -   Using pull requests for integrating changes
+    -   Managing tasks with GitHub issues
+3.  Recognise how Git and GitHub facilitate collaboration, focusing on:
+    -   Coordinating changes between multiple contributors
+    -   Tracking contributions and changes over time
+    -   Encouraging transparency and accountability in projects
```

Comment:
Related to using Bloom's taxonomy, "recognize" is a vague word. How will they recognize? How will they know they have recognized it?